k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -4402,6 +4395,25 @@ public boolean modeMismatch(Account.FolderMode aMode, Folder.FolderClass fMode)
         }
     }
 
+    private boolean isContact(Address[] addrs) {
```

Comment:
This method can probably replaced by `com.fsck.k9.helper.Contacts#isInContacts(String)`.